hardening/ansible/test.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+
+import os
+import re
+import subprocess
+
+from lib import util, results, virt, oscap, versions
+from conf import remediation_excludes
+
+
+virt.setup_host()
+
+profile = os.environ['PROFILE']
+profile_full = f'xccdf_org.ssgproject.content_profile_{profile}'
+
+use_gui = os.environ.get('USE_SERVER_WITH_GUI')
+
+if use_gui:
+    g = virt.Guest('gui_with_oscap')
+else:
+    g = virt.Guest('minimal_with_oscap')
+
+if not g.can_be_snapshotted():
+    ks = virt.Kickstart()
+    if use_gui:
+        ks.add_package_group('Server with GUI')
+    g.install(kickstart=ks)
+    g.prepare_for_snapshot()
+
+# TODO: no clue why this is needed, but it was in the old tests
```

Comment:
We have a function which documents the proper way how to install Ansible + additional modules: https://src.fedoraproject.org/tests/scap-security-guide/blob/main/f/Library/scap-common/lib.sh#_202

Basically, `ansible-core` in RHEL/CS doesn't contain all the modules which are used in the scap-security-guide  so we need to either install `rhc-worker-playbook` RPM (RHEL) or install those modules using `ansible-galaxy` (CentOS Stream).

Reply:
I switched the code to just use `rhc-worker-playbook` alone, per https://access.redhat.com/articles/remediation. If/when we need to run on CentOS Stream, we can easily uncomment it and add a condition.